ecommerce/certprep/templates/oscar/basket/partials/hosted_checkout_basket.html

```diff
@@ -61,17 +61,18 @@
                                 class="btn btn-success checkout-button">
                                     {% trans "Enroll" %}
                                 </a>
-                            {% else %}
-                                {% settings_value 'payment_support_email' as support_email %}
-                                {% settings_value 'payment_support_url' as support_url %}
-                                <p>{% trans "The enrollment link is invalid, please contact" %}
-                                    <a {% if support_email %} href=mailto:{{ support_email }} {% else %} href={{ support_url }} {% endif %}>
-                                        {% trans "Pearson VUE Support" %}
-                                    </a>.
-                                </p>
                             {% endif %}
                         </div>
                     </div>
+                    {% if not free_basket %}
+                        {% settings_value 'payment_support_email' as support_email %}
+                        {% settings_value 'payment_support_url' as support_url %}
+                        <p id="support-text">{% trans "The enrollment link is invalid, please contact" %}
```

Comment:
```suggestion
                        <p class="support-text">{% trans "The enrollment link is invalid, please contact" %}
```

What do you think about using a class name instead of an ID to style?

Reply:
Well, I'm using that element only once, so I figured it would be okay to use an id instead of a class name. But, I can change it if it's necessary.